### ISSUE TITLE
pretrained model results dont match published

### ISSUE THREAD

**Issue by naaz-git** [zijundeng/BDRAR#15]
I used pretrained model uploaded in the link and run inference for SBU test set.
These results do not match with the published results.
Why is such a case?

**Comment by hmQ1998** [comment]
哥们结果训练出来了吗

**Comment by naaz-git** [comment]
I simply took pretrained model from Authors link in description:

**Trained Model**
You can download the trained model which is reported in our paper at [Google Drive](https://drive.google.com/open?id=1Cw3nUmWEmnTnAVXPn3xZYhQ_uzmWmsr7).

Requirement
Python 2.7
PyTorch 0.4.0
torchvision...
